mobile/package.json

```diff
@@ -14,6 +14,10 @@
         "start:dev": "react-native start"
     },
     "dependencies": {
+        "@fortawesome/fontawesome-svg-core": "6.4.2",
+        "@fortawesome/free-brands-svg-icons": "6.4.2",
+        "@fortawesome/free-solid-svg-icons": "6.4.2",
+        "@fortawesome/react-native-fontawesome": "0.3.0",
```

Comment:
Let's use [https://github.com/oblador/react-native-vector-icons](https://github.com/oblador/react-native-vector-icons) instead

Reply:
I have checked fort awesome icons, free icons didn't contain all icons that we needed.  As for  react-native-vector-icons
this library seems like very large, maybe it's not that we need. 
Maybe it will be better to add svg icon instead?.. Waiting for others to discuss

Reply:
In the web app, we use material icons. That is why I think - https://github.com/oblador/react-native-vector-icons will be good

Reply:
fix